tests/Integrations/Guzzle/V6/GuzzleIntegrationTest.php

```diff
@@ -257,7 +257,7 @@ public function testLimitedTracerDistributedTracingIsPropagated()
             /** @var Tracer $tracer */
             $tracer = GlobalTracer::get();
             $tracer->setPrioritySampling(PrioritySampling::AUTO_KEEP);
-            $span = $tracer->startActiveSpan('custom')->getSpan();
+            $span = $tracer->startRootSpan('custom')->getSpan();
```

Comment:
I suspect users may not be using `startRootSpan` as they apparently should (it existed prior to this PR). We set priority sampling before flushing if it's missing, but this is probably too late? Doesn't it need to get propagated to all the downstream distributed traces?

Reply:
It's needed in the tests, but Bootstrap will do that for users normally. Nothing for them to change.

Reply:
I was thinking more about people who are creating spans from scratch in queue workers. It won't be set for them, yes?

Reply:
They shouldn't have autogenerated root spans, then it's not an issue either.

Reply:
It's really only a problem with testing when we use autogenerated root spans, but that's not impacting real world usage.